profiles/default/flink/build.gradle

```diff
@@ -31,6 +32,7 @@ dependencies {
     implementation "org.apache.flink:flink-connector-kafka:$kafkaVersion"
     implementation "org.apache.flink:flink-connector-jdbc:$jdbcVersion"
     implementation "org.apache.flink:flink-avro-confluent-registry:$flinkVersion"
+    implementation "org.apache.flink:flink-connector-postgres-cdc:$postgresCdcVersion"
```

Comment:
What if we make it possible to define flink dependencies in package json? Or we don't want to mix this into package.json?

Reply:
Yeah, likely, although I don't know the best way to go about it.

Reply:
Pease add an additional dependences section to the package.json which gets added to the build.gradle?